### ISSUE TITLE
bug: Cannot see `dataset` after enable PUBLIC_ENABLE_EXPERIMENTAL_FEATURES 

### ISSUE THREAD

**Issue by gyliu513** [langfuse/langfuse#337]
### Describe the bug

- When inspect langfuse server, we can see LANGFUSE_ENABLE_EXPERIMENTAL_FEATURES is set to true.

```yaml
"Env": [
                "NEXTAUTH_SECRET=mysecret",
                "NEXTAUTH_URL=http:localhost:3000",
                "TELEMETRY_ENABLED=true",
                "NEXT_PUBLIC_SIGN_UP_DISABLED=false",
                "LANGFUSE_ENABLE_EXPERIMENTAL_FEATURES=true",
                "NODE_ENV=production",
                "DATABASE_URL=postgresql://postgres:postgres@db:5432/postgres",
                "DIRECT_URL=postgresql://postgres:postgres@db:5432/postgres",
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "NODE_VERSION=18.18.0",
                "YARN_VERSION=1.22.19",
                "NEXT_TELEMETRY_DISABLED=1",
                "PORT=3000"
            ],
```

### To reproduce

set PUBLIC_ENABLE_EXPERIMENTAL_FEATURES as true when start docker compose

**Comment by marcklingen** [comment]
Hi @gyliu513, I deployed a fix in #335, have you tried using the latest container?

**Comment by gyliu513** [comment]
it works, thanks @marcklingen
